x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/CsvTests.java

```diff
@@ -455,7 +455,7 @@ private static void opportunisticallyAssertPlanSerialization(PhysicalPlan... pla
                 }
             } while (tmp.children().isEmpty() == false && (tmp = tmp.children().get(0)) != null);
 
-            SerializationTestUtils.assertSerialization(plan);
+            SerializationTestUtils.assertSerialization(plan, configuration);
```

Comment:
The config was _always_ important for serialization and subsequent deserialization to produce equal results; now our csv tests actually catch this: without using the correct `EsqlConfiguration`, differences are being detected in e.g. `date_format` (and rightly so).